src/jit/inlinepolicy.cpp

```diff
@@ -26,7 +26,19 @@
 
 InlinePolicy* InlinePolicy::GetPolicy(Compiler* compiler, bool isPrejitRoot)
 {
-    // For now, always create a Legacy policy.
+
+#ifdef DEBUG
+
+    // If JitInlineRandomSeed is set, use the RandomPolicy.
+    unsigned seed = JitConfig.JitInlineRandomSeed();
+    if (seed != 0)
+    {
+        return new (compiler, CMK_Inlining) RandomPolicy(compiler, isPrejitRoot, seed);
+    }
```

Comment:
I think my suggestion is same as what you propose to do with a very minor difference:

With your suggestion:
compInlineRandomStress = true with 100% probability (because MAX_STRESS_WEIGHT is 100).
compInlineStress = true with 0.5 probability

since the condition to select random policy is

compiler->compInlineRandomStress() && !compiler->compInlineStress()

essentially with 0.5 probability random policy gets selected.

Instead my suggestion is

```
useRandomPolicy = compStressCompile(STRESS_RANDOM_INLINE, 50);
if (useRandomPolicy)
{
// use random policy
}
else
{
   // use legacy policy
}

```
Note that random policy will remain in effect for all methods with the following settings alone with my suggestion:

```
set complus_jitstress=2020202
set complus_jitstressModeNames=STRESS_RANDOM_INLINE
```

Because JitSTressModeNames settings will make compStressCompile() return true irrespective of the weight parameter passed to it.